phpBB/includes/acp/acp_extensions.php

```diff
@@ -162,7 +162,8 @@ function main()
 				}
 				catch (\phpbb\extension\exception $e)
 				{
-					trigger_error($e . adm_back_link($this->u_action), E_USER_WARNING);
+					$message = call_user_func_array(array($this->user, 'lang'), array_merge(array($e->getMessage()), $e->get_parameters()));
```

Comment:
No more back link? (To be honest I never know when to add them and when not to...)